### ISSUE TITLE
UX Polish: "Show Codename" widget behavior on Submit page

### ISSUE THREAD

**Issue by ninavizz** [freedomofpress/securedrop#5239]
## Description

The recent change to re-position the "Show Codename" widget on the Submit page, was a terrific improvement. 

A behavioral particular with it that I just noticed, however, is that the `scrollto` piece feels jarring; and with the top-most portion of the Tor browser also sharing the same grey as the codename widget's background, usability is impaired. 

`scrollto` is a great functional asset, when needing to bring an area of a page into focus that is dozens to hundreds of pixels away. When implemented well, it will scroll to a div that positions the targeted content in the path of a user's vision—which is typically near where they clicked to bring the content into view, when initiating the interaction. 

TL;DR, for this issue—with the widget already being at the top—I'd prefer to simply nix that part of the functionality. I don't feel it is necessary in this context. The widget should simply "open" down. 

## User Stories

As a Source desiring to be reminded of what my codename is, I want to be able to click on "Show" and have my codename revealed in the existing gray-patch on the screen. When the gray patch moves-up to meld in with the Tor browser chrome, the little black letterforms are lost and I become confused. 

Video for demonstration: https://drive.google.com/open?id=1M5qx_Rnss2q1NlYqB7HO2z4hF6h5pNqO

**Comment by prateekj117** [comment]
@ninavizz @kushaldas I would like to work on this.
